style.css

```diff
@@ -0,0 +1,834 @@
+  width: 280px;
+  height: 10vh;
+}
+
+.tags1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 50px;
+  height: 7vh;
+  list-style: none;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.tags h6 {
+  width: 32px;
+  height: 20px;
+  list-style: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.tags2 {
+  display: flex;
+  list-style: none;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 95px;
+  height: 7vh;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.tags2 h6 {
+  width: 71px;
+  height: 20px;
+  list-style: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.tags3 {
+  display: flex;
+  list-style: none;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 61px;
+  height: 7vh;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.tags3 h6 {
+  width: 37px;
+  height: 20px;
+  list-style: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.but {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding-top: 0.4rem;
+  height: 40px;
+  width: 100%;
+  background: #ff6b00;
+  justify-content: center;
+  position: relative;
+}
+
+.see1 {
+  height: 40px;
+  width: 100%;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  text-decoration: none;
+  text-align: center;
+  align-items: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  padding: 2px 10px;
+  background: #1c1a19;
+  width: 100%;
+  left: 0;
+  margin-top: 2rem;
+}
+
+.about-sub {
+  padding: 1rem;
+  gap: 1rem;
+  display: flex;
+  flex-direction: column;
+}
+
+.primary {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-size: 60px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  width: 100%;
+  padding-top: 8rem;
+}
+
+.description {
+  width: 90%;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+  letter-spacing: 0.5px;
+  font-weight: 100;
+}
+
+.get {
+  padding: 12px;
+  width: 10rem;
+  height: 3rem;
+  background: #ff6b00;
+  align-items: center;
+  justify-content: center;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  text-decoration: none;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.img-about {
+  position: absolute;
+  width: 50%;
+  height: 39vh;
+  right: 0;
+  transform: matrix(-1, 0, 0, 1, 0, 0);
+}
+
+.img-about img {
+  position: absolute;
+  bottom: 2rem;
+  transform: matrix(-1, 0, 0, 1, 0, 0);
+}
```

Comment:
- [ ] Consider this as an optional change as previous reviewer might have missed:
- [ ] I suggest you to align this backgorund image to its correct postition in figma template: 
yours: 
![image](https://user-images.githubusercontent.com/91486791/205147232-9b238769-66b4-4bdb-8231-b6fbc6a697bd.png)
figma:
![image](https://user-images.githubusercontent.com/91486791/205147336-0c18e32c-543b-4455-8bef-10c7b50f58b0.png)